templates/includes/sidebar.html

```diff
@@ -1,4 +1,18 @@
-<ul class="sidebar-nav">
+<ul style="list-style: none">
+  {% if user.is_authenticated %}
+    <p>Hello,
+      <strong><a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}!</a></strong>
```

Comment:
The link to the user's profile uses the `driver-detail` URL pattern with `user.id`. Ensure that the `driver-detail` view is correctly set up to handle this URL and that the user is indeed a `Driver` instance.